SPECS/cpulimit/cpulimit.spec

```diff
@@ -0,0 +1,39 @@
+Name:		cpulimit
+Version:        1.1	
+Release:	1%{?dist}
+Summary:	CPU Usage Limiter for Linux
+
+Group:		Applications/System
+License:	GPLv2+
+URL:		http://cpulimit.sourceforge.net/
+Source0:	http://downloads.sourceforge.net/project/%{name}/%{name}/%{name}/%{name}-%{version}.tar.gz
+BuildRoot:	%(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX)
```

Comment:
Why do we need to set BuildRoot:	%(mktemp -ud %{_tmppath}/%{name}-%{version}-%{release}-XXXXXX) ?